frontend/src/pages/project/Snapshots/dialogs/ConfirmSnapshotRollbackDialog.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <ff-dialog :open="isOpen" header="Rollback Snapshot">
+        <template v-slot:default>
+            <p>This rollback will overwrite the current project.</p>
+            <p>All changes made since the last snapshot was taken will be lost.</p>
+            <p>Are you sure you want to rollback to this snapshot?</p>
```

Comment:
```suggestion
            <p>This rollback will overwrite the current project.</p>
            <p>All changes to the flows, settings and environment variables made since
                   the last snapshot will be lost.</p>
            <p>Are you sure you want to rollback to this snapshot?</p>
```